package.json

```diff
@@ -31,7 +31,8 @@
   },
   "dependencies": {
     "prop-types": "^15.6.1",
-    "react-native-animatable": "^1.2.4"
+    "react-native-animatable": "^1.2.4",
+    "react-native-extra-dimensions-android": "^0.21.0"
```

Comment:
Installing `react-native-extra-dimensions-android` requires linking it natively, which is an extra step that I'd like to not introduce in this module.  
I think a better way to handle this issue (until RN fixes it) would be moving `deviceWidth` and `deviceHeight` from the state to the props to make them easily customizable. What do you think?